src/main/java/org/cryptomator/fusecloudaccess/CloudAccessFS.java

```diff
@@ -28,17 +32,22 @@
 	private final OpenFileFactory openFileFactory;
 
 	public CloudAccessFS(CloudProvider provider, int timeoutMillis) {
+		this(provider,timeoutMillis,new OpenFileFactory(provider));
+	}
+
+	//Visible for testing
+	CloudAccessFS(CloudProvider provider, int timeoutMillis, OpenFileFactory openFileFactory){
 		this.provider = provider;
 		this.timeoutMillis = timeoutMillis;
-		this.openFileFactory = new OpenFileFactory(provider);
+		this.openFileFactory = openFileFactory;
 	}
 
 	/**
 	 * Method for async execution.
 	 *
-	 * @apiNote Only visible for testing.
 	 * @param returnCode an integer {@link CompletionStage} to execute
 	 * @return an integer representing one of the FUSE {@link ErrorCodes}
+	 * @apiNote Only visible for testing.
```

Comment:
Have you tried running `maven-javadoc-plugin` with this annotation?

Reply:
Yes. Runs through without errors.

Reply:
Strange. I always have problems with such tags, unless reconfiguring the javadoc plugin [as done in cryptofs](https://github.com/cryptomator/cryptofs/blob/d1be7a7e4cf5c24837556767b4596807c2843947/pom.xml#L223-L268).

Reply:
Then i'll remove this annotation for the time being. Compile compabillity on different systems is imho more important.

Reply:
If it works on one system, it will work on all. I was just wondering if you _tested_ it, as the javadoc isn't normally being built. I don't want it to fail when doing a release build.